juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -35,24 +35,52 @@ healthbot {
              */			
             field fan-name {
                 sensor fan-netconf {
-                    path Item;
+                    path name;
                 }
                 type string;
                 description "Name of the fan tray and fan number";
             }
-            field measurement {
+            field measurements {
```

Comment:
lets not change the field names as this will impact UI and test

Reply:
measurement field is still present and uses capture-group formula to derive integer value from measurements.
Measurements is new field which is a string type.